app/models/article.rb

```diff
@@ -648,13 +644,16 @@ def evaluate_markdown
   end
 
   def extracted_evaluate_markdown
-    return unless processed_content
+    content_renderer = processed_content
+    return unless content_renderer
+
+    self.processed_html = content_renderer.process(calculate_reading_time: true)
+    self.reading_time = content_renderer.reading_time
 
-    self.reading_time = processed_content.calculate_reading_time
-    self.processed_html = processed_content.finalize
+    front_matter = content_renderer.front_matter
 
     if front_matter.any?
-      evaluate_front_matter
+      evaluate_front_matter(front_matter)
```

Comment:
Changed back to passing the parameter explicitly for (explicity and) consistency with the prev version. We can change the code when we remove the feature flag.